### ISSUE TITLE
Get-FolderSizeInfo reports incomplete data

### ISSUE THREAD

**Issue by jdhitsolutions** [jdhitsolutions/PSScriptTools#60]
The Get-FolderSizeInfo command isn't returning the information as expected.

```powershell
 dir c:\scripts -file -Recurse | measure length -sum | select count,sum

Count       Sum
-----       ---
 7537 538844457
```

should be the same as this:

```powershell
Get-FolderSizeInfo -Path c:\scripts

Computername    Path                                                TotalFiles     TotalSize
------------    ----                                                ----------     ---------
BOVINE320       C:\scripts                                                3479     132304770
```

And that's not even taking hidden files into account.

**Comment by jdhitsolutions** [comment]
I also discovered I was originally using a .NET class that didn't exist in Windows PowerShell. Modified the function to use different code depending on the PowerShell version.

**Comment by jdhitsolutions** [comment]
Fixed in v2.15.1
